### ISSUE TITLE
plot() function error

### ISSUE THREAD

**Issue by paulinetrinh** [adw96/breakaway#49]
When using the breakaway plot function of alpha estimates the following ggplot2 error occurs: 

Error: Aesthetics must be either length 1 or the same as the data (26): x, y, colour, shape

**Comment by adw96** [comment]
This works fine:

`library(phyloseq)
data(GlobalPatterns)
library(breakaway)
ba <- breakaway(GlobalPatterns)
plot(ba)
`

??

**Comment by adw96** [comment]
Ok, this now fails, TODO(Amy)

Fix may address issue #50 as well

```
library(phyloseq)
data(GlobalPatterns)
library(breakaway)
ba <- breakaway(GlobalPatterns)
# plot(ba)

plot(ba, GlobalPatterns, color = "SampleType")

plot(ba, GlobalPatterns)
```

**Comment by adw96** [comment]
Closed with 1d71024
